src/ethereum/cancun/vm/instructions/memory.py

```diff
@@ -138,3 +140,39 @@ def msize(evm: Evm) -> None:
 
     # PROGRAM COUNTER
     evm.pc += 1
+
+
+def mcopy(evm: Evm) -> None:
+    """
+    Copy the bytes in memory from one location to another.
+
+    Parameters
+    ----------
+    evm :
+        The current EVM frame.
+
+    """
+    # STACK
+    dst = pop(evm.stack)
+    src = pop(evm.stack)
```

Comment:
```suggestion
    destination = pop(evm.stack)
    source = pop(evm.stack)
```

Should probably avoid the abbreviations.

Reply:
Updated.